spec/lib/annotate/annotate_models_spec.rb

```diff
@@ -1831,9 +1831,9 @@ class Foo < ActiveRecord::Base
   describe 'annotating a file' do
     before do
       @model_dir = Dir.mktmpdir('annotate_models')
-      (@model_file_name, @file_content) = write_model 'user.rb', <<-EOS
-class User < ActiveRecord::Base
-end
+      (@model_file_name, @file_content) = write_model 'user.rb', <<~EOS
```

Comment:
I looked up what the differences are in the doc string `<<-` vs `<<~`. Learning something new 👍

Reply:
cf. https://www.rubyguides.com/2018/11/ruby-heredoc/

I like `<<~` because it makes code more clean.
When using VS Code, both strings indented with `<<~` and that of `<<-` can be collapsed, but the latter looks bad.

Reply:
Yep the `<<~` does look more clean. I think that it may have not existed when that spec was written but now it does 😊